tests/functional/verification/test_pset.py

```diff
@@ -0,0 +1,35 @@
+from nagini_contracts.contracts import *
+
+
+class A:
+    pass
+
+
+def test_pset() -> None:
+    ints = PSet(1, 2, 3)
+    four = PSet(4)
+    a = A()
+    ass = PSet(a)
```

Comment:
Please, choose a different name for this variable.

Reply:
Lol